client/src/components/Stars.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const Stars = (props) => {
+    let starts_filling_percentage = Math.round((props.rate/props.reviews_number)*20);
```

Comment:
I'm not sure what `*20` is for. Is it equal to `(1/5) * 100`? Maybe you can rewrite it with a constant like `(1/NUM_STARS) * 100` where `NUM_STARS = 5`.

Reply:
props.rate is basically the summation of rate that will be passed to this component so to callculate the percentage i divide the summation / total  and i multiply by 20 cause cause the max rate is 5 stars

Reply:
OK I was just suggesting that you replace the number 20 with a variable name or expression so it's more clear to people who read the code. But I get what you mean.